app/controllers/api/base_controller/normalizer.rb

```diff
@@ -56,6 +56,10 @@ def normalize_attr(attr, value)
           Float::INFINITY.to_s
         elsif Api.resource_attribute?(attr)
           normalize_resource(value)
+        elsif value == NilClass
+          {}
```

Comment:
@kbrock do you have any idea why this was needed?  The test below was expecting a nil v_workflow_class to come back as `{}`. Not a string, not json `null`, but `{}`.